src/index.tsx

```diff
@@ -15,13 +14,14 @@ const sw = await navigator.serviceWorker.register(new URL('sw.ts', import.meta.u
 const root = ReactDOMClient.createRoot(container)
 
 // SW did not trigger for this request
-if (isPathOrSubdomainRequest(BASE_URL, window.location)) {
+if (isPathOrSubdomainRequest(window.location)) {
   // but the requested path is something it should, so show redirect and redirect to the same URL
   root.render(
     <RedirectPage />
   )
   window.location.replace(window.location.href)
 } else {
+  // TODO: add detection of DNSLink gateways (alowing use with Host: en.wikipedia-on-ipfs.org)
```

Comment:
if `en.wikipedia-on-ipfs.org.ipns.sw-host.tld` is requested, this will never be hit.